tools/biobb_pytorch/biobb_apply_mdae.xml

```diff
@@ -0,0 +1,92 @@
+<tool id="biobb_pytorch_apply_mdae" name="ApplyMdae" version="4.2.1" >
+    <description>Apply a Molecular Dynamics AutoEncoder (MDAE) PyTorch model.</description>
+
+    <requirements>
+      <requirement type="package" version="4.2.1">biobb_pytorch</requirement>
+    </requirements>
+
+    <command detect_errors="exit_code"><![CDATA[
+
+      ln -s $input_data_npy_path ./input_data_npy_path.$input_data_npy_path.ext;
+      ln -s $input_model_pth_path ./input_model_pth_path.$input_model_pth_path.ext;
+      #if str($config_json) != 'None':
+        ln -s $config_json ./config_json.$config_json.ext;
+      #end if
+
+      apply_mdae
+
+      #if str($config_json) != 'None':
+        --config ./config_json.$config_json.ext
+      #end if
+
+      --input_data_npy_path ./input_data_npy_path.$input_data_npy_path.ext
+      --input_model_pth_path ./input_model_pth_path.$input_model_pth_path.ext
+      --output_reconstructed_data_npy_path $outname_output_reconstructed_data_npy_path
+      #if str($outname_output_latent_space_npy_path) != 'None':
+        --output_latent_space_npy_path $outname_output_latent_space_npy_path
+      #end if
+      ;
+
+      if test -f $outname_output_reconstructed_data_npy_path; then mv $outname_output_reconstructed_data_npy_path $output_reconstructed_data_npy_path; fi;
+      if test -f $outname_output_latent_space_npy_path; then mv $outname_output_latent_space_npy_path $output_latent_space_npy_path; fi;
+
+      ]]>
+    </command>
+
+    <inputs>
+      <param name="input_data_npy_path"  type="data" format="npy"  optional="False" label="input NPY file" help="Path to the input data file. Format: [input].npy"/>
```

Comment:
```suggestion
      <param name="input_data_npy_path"  type="data" format="npy"  optional="False" label="Input NPY file" help="Input data file"/>
```

I think referring to a path in the GUI doesn't make as much sense as in a CLI (same for the other params below)